JotunnLib/Configs/RoomConfig.cs

```diff
@@ -0,0 +1,92 @@
+using Jotunn.Entities;
+
+namespace Jotunn.Configs
+{
+    /// <summary>
+    ///     Configuration class for adding custom clutter.<br />
```

Comment:
you missed to replace clutter :P

Reply:
Addressed in: https://github.com/Valheim-Modding/Jotunn/pull/434/commits/5991d7793bc68f46effaa47970290fd1f5e6f39f